libraries/AP_AHRS/AP_AHRS.cpp

```diff
@@ -2277,7 +2277,7 @@ bool AP_AHRS::attitudes_consistent(char *failure_msg, const uint8_t failure_msg_
             // check roll and pitch difference
             const float rp_diff_rad = primary_quat.roll_pitch_difference(ekf2_quat);
             if (rp_diff_rad > ATTITUDE_CHECK_THRESH_ROLL_PITCH_RAD) {
-                hal.util->snprintf(failure_msg, failure_msg_len, "EKF2 Roll/Pitch inconsistent by %d deg", (int)degrees(rp_diff_rad));
+                hal.util->snprintf(failure_msg, failure_msg_len, "EKF2 Roll/Pitch inconsistent with current AHRS. Wait or check sensors");
```

Comment:
"EKF2 Roll/Pitch inconsistent by %d deg. Wait or reboot", (int)degrees(rp_diff_rad)

Reply:
we could replace "inconsistent" with "diff"

Reply:
done